index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link rel="stylesheet" href="stylesheet.css">
+  </head>
+  <body>
+    <div class="maincont">
+
+      <div class="app-bar">
+        <div class="upper-bar">
+          <p class="time">9 : 41</p> 
+          <div class="upper-icons">
+            <p class="ntew"> </p>
+            <p class="wifi"></p>
+            <p class="battery"></p>
+          </div>
+        </div>
+        <div class="lower-bar">
+          <p class="logoo"><img src="image/Logo.png" alt="logo icon"></p> 
+          <div class="menu-icons"> <img class="menuimg" src="image/menu.png" alt="menu icon"></div>
+        </div>
+      </div>
+
+      <div class="bgimage">
+        <p class="myname">Somarib Emmanuel</p>
```

Comment:
- [ ] Kindly change the `p` with a class of `myname` to a `h1` tag 🙏 . This is the main title of your site and would allow your webpage to be easily picked up by google and search engines.